src/ai/backend/manager/models/alembic/versions/308bcecec5c2_add_groups_type.py

```diff
@@ -0,0 +1,153 @@
+        onupdate=sa.func.current_timestamp(),
+    ),
+    #: Field for synchronization with external services.
+    sa.Column("integration_id", sa.String(length=512)),
+    sa.Column(
+        "domain_name",
+        sa.String(length=64),
+        sa.ForeignKey("domains.name", onupdate="CASCADE", ondelete="CASCADE"),
+        nullable=False,
+        index=True,
+    ),
+    # TODO: separate resource-related fields with new domain resource policy table when needed.
+    sa.Column("total_resource_slots", JSONB(), default={}),
+    sa.Column(
+        "allowed_vfolder_hosts",
+        JSONB(),
+        nullable=False,
+        default={},
+    ),
+    # dotfiles column, \x90 means empty list in msgpack
+    sa.Column(
+        "dotfiles", sa.LargeBinary(length=MAXIMUM_DOTFILE_SIZE), nullable=False, default=b"\x90"
+    ),
+    sa.Column(
+        "resource_policy",
+        sa.String(length=256),
+        sa.ForeignKey("project_resource_policies.name"),
+        nullable=False,
+    ),
+    sa.Column(
+        "type",
+        EnumValueType(ProjectType),
+        nullable=False,
+        default=ProjectType.GENERAL,
+    ),
+    sa.UniqueConstraint("name", "domain_name", name="uq_groups_name_domain_name"),
+)
+association_groups_users = sa.Table(
+    "association_groups_users",
+    metadata,
+    sa.Column(
+        "user_id",
+        GUID,
+        sa.ForeignKey("users.uuid", onupdate="CASCADE", ondelete="CASCADE"),
+        nullable=False,
+        primary_key=True,
+    ),
+    sa.Column(
+        "group_id",
+        GUID,
+        sa.ForeignKey("groups.id", onupdate="CASCADE", ondelete="CASCADE"),
+        nullable=False,
+        primary_key=True,
+    ),
+)
+
+
+projecttype_choices = list(map(lambda v: v.value, ProjectType))
+projecttype = postgresql.ENUM(
+    *projecttype_choices,
+    name="projecttype",
+)
+
+
+def upgrade():
+    conn = op.get_bind()
+    projecttype.create(conn)
+    op.add_column(
+        "groups",
+        sa.Column("type", sa.Enum(*projecttype_choices, name="projecttype"), default="general"),
+    )
+    conn.execute(text("UPDATE groups SET type = 'general'"))
+    op.alter_column("groups", "type", nullable=False)
+
+    model_store_gid = uuid4()
+    conn.execute(
+        sa.insert(groups).values(
+            {
+                "id": model_store_gid,
+                "name": "model-store",
+                "domain_name": "default",
+                "resource_policy": "default",
+                "type": ProjectType.MODEL_STORE,
+            }
+        )
+    )
+
+    uids = conn.execute(sa.select([users.c.uuid]).where(users.c.domain_name == "default")).all()
+    conn.execute(
+        sa.insert(association_groups_users).values(
+            user_id=sa.bindparam("user_id"),
+            group_id=model_store_gid,
+        ),
+        [{"user_id": uid[0]} for uid in uids],
+    )
+
+
+def downgrade():
+    conn = op.get_bind()
+    conn.execute(text("DELETE FROM groups WHERE name = 'model-store'"))
```

Comment:
This should refer the `type` column, _not_ the `name` column.
(FYI: To use a keyword like `type`, `user`, etc. as a column name in SQL, surround it with double-quotes.)